pytrendy/post_processing/segments_refine/gradual_expand_contract.py

```
@@ -88,3 +88,57 @@ def _get_window_df(center: str, days: int = 7) -> pd.DataFrame:
+        overlaps = meta_df.loc[(meta_df['start'] > gradual_end) & (meta_df['start'] <= new_end)]
+        overlaps_nonflats = overlaps[overlaps['direction'] != 'Flat']
+
+        if not overlaps_nonflats.empty:
+            first_notflat_overlap = overlaps_nonflats.iloc[0]
+            new_end = pd.to_datetime(first_notflat_overlap['start']) - pd.Timedelta(days=1)
+
+        new_end = min(new_end, df.index[-1])
+        segments_padded[i]['end'] = new_end.strftime('%Y-%m-%d')
+        update_next_segment(i, new_end, segments, segments_padded)
+
+        segments_padded[i]['padded'] = True if new_end != gradual_end else False
```

Ruff (0.15.21)

[warning] Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)